### PR TITLE
Add robustnerf mask to mipnerf360

### PR DESCRIPTION
**Pull request by Sarasra**
To enable robustnerf the flag config.data_loss_type should be set to 'robustnerf'. If enabled the create_robustnerf_mask creates a mask that is multiplied to data_loss to enable ignoring parts of the scene. The mask is created based on a revolving percentile of the loss. Since the loss only gets aggregated in the main training loop, the current percentile is recorded in the stats and the aggregated loss_threshold is then passed to the train_steps.
The implementation is based on "RobustNerf: ignoring distractors with robust optimization".
This is a work in progress.

Passes all the tests in the default setting.

### CODE REVIEW

**Comment by martinarroyo** [comment]
Just FYI there is a typo in config.py: `enable_robutnerf_loss` should be `enable_robustnerf_loss`. The other instances of this config flag seem fine though.

**Comment by Sarasra** [comment]
Thanks for the catch. I'll update shortly with robustnerf functionality updated.
